networking/handshake_handlers.go

```diff
@@ -374,7 +374,7 @@ func (nm *Handshake) SendVersion(peer salticidae.PeerID) error {
 	build := Builder{}
 	v, err := build.Version(nm.networkID, nm.clock.Unix(), toIPDesc(nm.myAddr), ClientVersion)
 	if err != nil {
-		return fmt.Errorf("packing Version failed due to %s", err)
+		return fmt.Errorf("packing version failed due to: %w", err)
```

Comment:
It seems like we're keeping the messages capitalized in the other messages, so this should either stay capitalized, or the rest should be made lowercase